pubtools/_ami/tasks/push.py

```diff
@@ -448,6 +451,14 @@ def add_args(self):
             default=4,
         )
 
+        group.add_argument(
+            "--snapshot_accounts",
```

Comment:
The name --snapshot_accounts is inconsistent with all other argument names which always use - and not _ to separate multiple words.  Can you please use - instead of _ to be consistent with other args?

Actually, there is also an 'accounts' argument but unfortunately the syntax looks totally different to this one.  It seems confusing for --accounts and --snapshot-accounts arguments to take args of completely different format, so it might be a good idea to make this --snapshot-account-ids instead to clarify that it is only IDs being passed.